src/strategies/LiquityStabilityPoolStrategy.sol

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import "BoringSolidity/interfaces/IERC20.sol";
+import "BoringSolidity/libraries/BoringERC20.sol";
+import "interfaces/ILiquityStabilityPool.sol";
+import "./BaseStrategy.sol";
+
+contract LiquityStabilityPoolStrategyFrontendTag {
+    constructor(ILiquityStabilityPool _pool) {
+        _pool.registerFrontEnd(1e18);
+    }
+}
+
+contract LiquityStabilityPoolStrategy is BaseStrategy {
+    using BoringERC20 for IERC20;
+
+    error InvalidFeePercent();
+    error InsupportedToken(IERC20 token);
+    error SwapFailed();
+    error InsufficientAmountOut();
+
+    event FeeChanged(uint256 previousFee, uint256 newFee, address previousFeeCollector, address newFeeCollector);
+    event SwapperChanged(address oldSwapper, address newSwapper);
+    event RewardSwapped(IERC20 token, uint256 total, uint256 amountOut, uint256 feeAmount);
+    event RewardTokenUpdated(IERC20 token, bool enabled);
+
+    ILiquityStabilityPool public immutable pool;
+    address public immutable tag;
+
+    address public feeCollector;
+    uint8 public feePercent;
```

Comment:
why do we use a low precision here?

Reply:
the fees are from 0 to 100, it's not precise but I thought that we wouldn't need more precision here